packages/loader/container-loader/src/container.ts

```diff
@@ -1742,18 +1742,23 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this.messageCountAfterDisconnection = 0;
         }
 
-        const state = this.connectionState === ConnectionState.Connected;
-        if (!this.context.disposed) {
-            this.context.setConnectionState(state, this.clientId);
-        }
-        assert(this.protocolHandler !== undefined, 0x0dc /* "Protocol handler should be set here" */);
-        this.protocolHandler.quorum.setConnectionState(state, this.clientId);
-        raiseConnectedEvent(this.logger, this, state, this.clientId);
+        PerformanceEvent.timedExec(
+            this.logger,
+            { eventName: "OpsSentOnReconnect" },
```

Comment:
This event will fire when disconnecting. I'm not sure if we need it, and if we do, that the name corresponds to what actually happens here.